module/VuFind/src/VuFind/Connection/OauthServiceInterface.php

```diff
@@ -32,6 +32,8 @@
 /**
  * Interface for classes using OauthServiceTrait.
  *
+ * Classes which use this trait should also use LoggerAwareTrait.
```

Comment:
Should we mention the interface here rather than the trait?

Reply:
Agree, done.